src/components/ListUser.vue

```diff
@@ -41,39 +45,51 @@
             <td class="col-md-2 col-xs-2">{{user.phone}}</td>
             <td class="col-md-4 col-xs-4">
               <span v-if="!user.deleted">
-                <button class="btn btn-primary" @click="editUser($event, user)">Edit</button>
-                <button class="btn btn-danger" @click="toggleDeleteUser($event, user.id)">Delete</button>
+                <router-link to="/form-edit-user">
+                  <button class="btn btn-primary" @click="editUser($event, user)">Edit</button>
+                </router-link>
+                <button class="btn btn-danger" @click="beforeDeleteUser(user)">Delete</button>
               </span>
               <span v-else>
-                <button class="btn btn-info" @click="toggleDeleteUser($event, user.id)">Undo</button>
+                <button class="btn btn-info" @click="toggleDeleteUser(user.id)">Undo</button>
               </span>
-              <button class="btn btn-info" @click="changeShowTaskUser($event, user.id)">Task</button>
+              <router-link :to="'/user/' + userId + '/task'">
```

Comment:
Is there another way to manage `router-link to` instead of appending string? You can see `named router` section in vue-router document